run-windows.ps1

```diff
@@ -0,0 +1,157 @@
+# set -e #break script on non-zero exitcode from any command
+# set -x #display command being executed
+
+# startXvfb () {
+#     Xvfb $DISPLAY -screen 0 640x480x8 -nolisten tcp > /dev/null 2>&1 &
+#     XVFBPID=$!
+#     sleep 5
+# }
+
+# killXvfb () {
+#     if [ -n "${XVFBPID:-}" ]; then
+#         # Stop virtual X display server
+#         kill $XVFBPID
+#     fi
+# }
+
+$Env:BOOST_ROOT = "boost_${Env:BOOST_VERSION}".Replace(".","_")
+#$Env:GMOCK_PATH = "$(Get-Location | Select -Expand Path)/googletest"
+#$Env:GMOCK_ROOT = "$Env:GMOCK_PATH"
+
+# --- Function definitions ---
+function Invoke-CMake {
+    param (
+        $Arguments
+    )
+    $process = Start-Process "cmake" -ArgumentList "$Arguments" -NoNewWindow -Wait -PassThru
+    If ($process.ExitCode -ne 0) {
+        Write-Host "Abort script execution due to CMake error."
+        Exit(42)
+    }
+}
+
+function Get-VariableOrEnv {
+    param (
+        [string]$Name
+    )
+
+    $envValue = [Environment]::GetEnvironmentVariable($Name)
+    if ($envValue -ne $null) {
+        return $envValue
+    }
+
+    if (Get-Variable -Name $Name -ErrorAction SilentlyContinue) {
+        return (Get-Variable -Name $Name).Value
+    }
+
+    return $null
+}
+
+# --- Variable definitions ---
+$BUILD_SHARED_LIBS = Get-VariableOrEnv -Name "BUILD_SHARED_LIBS"
+$CMAKE_PREFIX_PATH = Get-VariableOrEnv -Name "CMAKE_PREFIX_PATH"
+$COVERALLS_SERVICE_NAME = Get-VariableOrEnv -Name "COVERALLS_SERVICE_NAME"
+$VALGRIND_TESTS = Get-VariableOrEnv -Name "VALGRIND_TESTS"
+$GMOCK_PATH = Get-VariableOrEnv -Name "GMOCK_PATH"
+$GMOCK_VER = Get-VariableOrEnv -Name "GMOCK_VER"
```

Comment:
Have remove unused variables. Would like to have similar behavior to Linux, where there is not explicit distinction between $Env: variables and "normal" variables. Thus the function. Otherwise, one would always need to check whether $Env: was set or not.